src/types/local_config.go

```diff
@@ -1,6 +1,6 @@
 package types
 
-// LocalConfig represents development specific configuration for an application
-type LocalConfig struct {
```

Comment:
I think this should still be called the `LocalConfig`. Why was it renamed?

Reply:
Because it didn't contain anything other than dependencies, but I can change it back